{
  "Header": {
    "Original": "CVE-2025-4287 (A vulnerability was found in PyTorch 2.6.0+cu124. It has been rated as ...)",
    "ID": "CVE-2025-4287",
    "Description": "(A vulnerability was found in PyTorch 2.6.0+cu124. It has been rated as ...)"
  },
  "Annotations": [
    {
      "Original": "- pytorch \u003cunfixed\u003e (bug #1104931)",
      "Type": "package",
      "Package": "pytorch",
      "Kind": "unfixed",
      "BugNo": 1104931
    },
    {
      "Original": "[bookworm] - pytorch \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "bookworm",
      "Package": "pytorch",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/pytorch/pytorch/issues/150836",
      "Type": "NOTE",
      "Description": "https://github.com/pytorch/pytorch/issues/150836"
    },
    {
      "Original": "NOTE: https://github.com/pytorch/pytorch/pull/150923",
      "Type": "NOTE",
      "Description": "https://github.com/pytorch/pytorch/pull/150923"
    }
  ]
}